pkg/controller/resourceclaim/metrics/metrics.go

```diff
@@ -53,6 +53,14 @@ var (
 			Help:           "Number of ResourceClaims",
```

Comment:
@pohly I think these metrics should be updated and condensed into two labeled metrics:

`resourceclaim_controller_creates_total`: counter with labels `adminaccess` (`true` or `false`), `status` (`fail` or `success`)
`resourceclaim_controller_resource_claims`: gauge with labels `adminsaccess` (`true` or `false`), `allocated` (`true` or `false`)

WDYT?

cc @kubernetes/sig-instrumentation-leads

Reply:
Happy to make the changes after we get consensus.

Reply:
Yes, that makes sense.

Reply:
With `resourceclaim_controller_creates_total: counter with label status (failure or success)`, operators will see the failure go up but not sure why. 

We had suggested this as part of the KEP moving the feature to beta. 

> Metrics in kube-controller-manager about total (resourceclaim_controller_resource_claims_adminaccess) and allocated ResourceClaims with adminAccess (resourceclaim_controller_allocated_resource_claims_adminaccess).

Without an adminaccess label for resourceclaim_controller_creates_total and allocated_resource_claims OR new metrics, then we do not have any metric specifically tracking failure and success for this feature.

Reply:
Ack, I agree with the use case for it.